contracts/modules/TransferManager/GTM/GeneralTransferManager.sol

```diff
@@ -171,7 +171,9 @@ contract GeneralTransferManager is GeneralTransferManagerStorage, TransferManage
                 txReq = transferRequirements[uint8(TransferType.GENERAL)];
             }
 
-            (canSendAfter, fromExpiry, canReceiveAfter, toExpiry) = _getValuesForTransfer(_from, _to);
+            !_isValidModule(_from) ? (canSendAfter, fromExpiry) = _getValuesForFrom(_from) : (canSendAfter, fromExpiry) = (uint64(now - 1), uint64(now + 1));
```

Comment:
Doesn't really matter, but maybe use (0, and maxUint64) instead here?

Reply:
How 0 will work for `canSendAfter & canRecieveAfter`? but yeah maxUint64 will work for expiry